lib/r10k/source/base.rb

```diff
@@ -23,16 +23,27 @@ class R10K::Source::Base
   # @param options [Hash] An additional set of options for this source. The
   #   semantics of this hash may depend on the source implementation.
   #
-  # @option options [Boolean] :prefix Whether to prefix the source name to the
+  # @option options [Boolean, String] :prefix Whether to prefix the source name or prefix name to the
```

Comment:
Unclear. Should probably read
> If a String, the prefix name for the environment directory names. Otherwise, whether to use the source name as the prefix. All sources should respect this option.